### PR TITLE
Add admin functions for wiping a version

### LINKED ISSUE #3684: Add admin functions for wiping a version from web instances
This is exposed to project owners, but an admin function here would be really helpful for debugging purposes. The existing view for wiping versions will likely need to be broken out to a reusable view for the admin function.

### PR DESCRIPTION
**Pull request by dojutsu-user**
Closes #3684

### CODE REVIEW

File: readthedocs/core/utils/general.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+from django.shortcuts import get_object_or_404
+
+from readthedocs.core.utils import broadcast
+from readthedocs.projects.tasks import remove_dirs
+from readthedocs.builds.models import Version
+
+
+def wipe_version_via_slug(version_slug):
+    """Wipes the given version."""
+    version = get_object_or_404(Version, slug=version_slug)
```
Comment (ericholscher) [inline readthedocs/core/utils/general.py]: This needs to be filtering by a project, not just a version slug.

Comment (dojutsu-user) [reply readthedocs/core/utils/general.py]: Okay.
I have pushed the changes.

**Comment by dojutsu-user** [comment]
@ericholscher 
I'm unable to reproduce the error in my local.
Also, I have pushed the changes suggested by you.

**Comment by dojutsu-user** [comment]
@ericholscher 
I have added one more test.
Can this PR get another round of review?

**Comment by ericholscher** [comment]
Thanks!

**Comment by 0bliqu3s** [comment]
dojutsu-user:admin-action-for-version #9683 #9682 #

**Pull request merged**